src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -70,6 +70,7 @@
     toggle-expand="toggleExpandPanel"
     create-child-ui-state="createChildUiState"
     toggle-expand="toggleExpandPanel"
+    reporting-items-count="{{state.panels.length}}"
```

Comment:
wouldn't just counting `reporting-item`s be enough ? i am worried we might at some point have other places with multiple visualizations and this adds another bit to think of without adding much value.